types/node/v16/globals.d.ts

```diff
@@ -87,6 +87,7 @@ interface RelativeIndexable<T> {
 }
 interface String extends RelativeIndexable<string> {}
 interface Array<T> extends RelativeIndexable<T> {}
+interface ReadonlyArray<T> extends RelativeIndexable<T> {}
```

Comment:
Not sure if any of these interfaces actually still need to exist in the `v16/globals.d.ts`, let alone whether this file should even still exist (since it looks like other Node 16 stuff is now in the main `globals.d.ts`), but I don't know the history here, so I'm taking the safe approach in this PR and just adding the missing declaration to both files.